src/azure-cli/azure/cli/command_modules/redis/_params.py

```diff
@@ -51,6 +51,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('mi_user_assigned', arg_type=user_identity_type)
         c.argument('update_channel', arg_type=get_enum_type(UpdateChannel), help='Specifies the update channel for the monthly Redis updates your Redis Cache will receive. Caches using "Preview" update channel get latest Redis updates at least 4 weeks ahead of "Stable" channel caches. Default value is "Stable".')
         c.argument('storage_subscription_id', options_list=['--storage-subscription-id', '--storage-sub-id'], help='SubscriptionId of the storage account')
+        c.argument('disable_access_key_authentication', options_list=['--disable-access-keys'], help='Authentication to Redis through access keys is disabled when set as true. Default value is false.')
```

Comment:
This argument is `str` type. Do you want it to be `bool` type?

Reply:
If you want to specify default value, add parameter `default='false'` for `c.argument()`

Reply:
Yes we want the argument to be boolean, using `get_three_state_flag` now. We don't need CLI to set the default value as false, so I've removed that part from its help text to avoid confusion